app/models/concerns/authorizable.rb

```diff
@@ -16,7 +16,7 @@ module Authorizable
       if user.nil?
         self.where('1=0')
       elsif user.admin?
-        self.scoped
+        self.where(nil)
```

Comment:
[This line](https://github.com/Katello/katello/blob/master/test/controllers/api/v2/systems_bulk_actions_controller_test.rb#L124) is the culprit. I'll submit a fix in Katello then we can revisit this.